Sources/Apollo/ApolloStore.swift

```diff
@@ -98,7 +98,7 @@ public final class ApolloStore {
 
       return try transaction.execute(selections: Query.Data.selections, onObjectWithKey: Query.rootCacheKey, variables: query.variables, accumulator: zip(mapper, dependencyTracker))
     }.map { (data: Query.Data, dependentKeys: Set<CacheKey>) in
-      GraphQLResult(data: data, errors: nil, dependentKeys: dependentKeys)
+      GraphQLResult(data: data, errors: nil, dataSource:.cache, dependentKeys: dependentKeys)
```

Comment:
I prefer `source` over `dataSource` here (and everywhere else), with a corresponding change of `GraphQLDataSource` to `GraphQLResult.Source`.